### ISSUE TITLE
Validate not working when type is an array

### ISSUE THREAD

**Issue by johnedvard** [Textalk/angular-schema-form#632]
Hi.
I have posted a question on [stackoverflow](http://stackoverflow.com/questions/35036463/how-to-validate-string-and-number-using-json-schema) because I am not able to make [my example](http://schemaform.io/examples/bootstrap-example.html#/cc9a172222514d43577d) work (being validated correctly).
when I post my schema and model in the [json-schema-validator](http://json-schema-validator.herokuapp.com/), my example work. Why is that? Am I doing something wrong in the form?

**Comment by nicklasb** [comment]
Yes, it seems like that works there. But I cannot find anywhere in the specification that having multiple values for "type" is supported, albeit a cursory look.
The validator for ASF is TV4, perhaps they have some information?

(I am not sure I would use that functionality even if I could, that'd just be moving a problem further along)

**Comment by nicklasb** [comment]
Oops, found it. Indeed it can be an array.

**Comment by Anthropic** [comment]
A quick look at source shows that array type definition is primarily supported to support ["null","non-null"], given the example the result could only be a string as that is what the form is set to. So given the form has to be set to something it makes supporting opposite types quite difficult which is why I suspect it hasn't been tackled by anyone.

**Comment by nicklasb** [comment]
Yes, it is probably for the same reason "anyOf" isn't supported yet.
